web/src/features/map/MapTooltip.tsx

```diff
@@ -39,11 +39,21 @@ function TooltipInner({
 
   const [currentMode] = useAtom(productionConsumptionAtom);
   const isConsumption = currentMode === Mode.CONSUMPTION;
-  const fossilFuelPercentage = getFossilFuelPercentage(
+  const intensity = getCarbonIntensity(
+    isConsumption,
+    co2intensity,
+    co2intensityProduction
+  );
+  const fossilFuelPercentage = getFossilFuelRatio(
     isConsumption,
     fossilFuelRatio,
     fossilFuelRatioProduction
   );
+  const renewable = getRenewableRatio(
+    isConsumption,
+    renewableRatio,
```

Comment:
Would have been nice for all of these if we named the data field renewableRationConsumption here

Reply:
Agreed, I can add it to this PR if you wish, should be a simple change of renaming the symbols.